### PR TITLE
STORE-1315: Redesign advanced search UI

### PR DESCRIPTION
**Pull request by Malintha**
https://wso2.org/jira/browse/STORE-1315

### CODE REVIEW

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-			{{> primary-search-attributes .}}
-			{{> extended-search-attributes .}}
-			{{> advanced-search-buttons .}}
-		</form>
-	</div>
-</div>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-lg-12 advanced-search-form-container">
+            <h1 >Advanced Search</h1>
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs]: missing "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+            <h2 class="advanced-search-field-title">
+                <a href="#collapseoverview" aria-expanded="false" class="collapsing-h2">
+                    Find assets in all asset types
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs]: missing "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-buttons.hbs [hunk 3]
```diff
@@ -1 +1,5 @@
-	<button id='search-btn' type="submit" class="btn btn-primary">SEARCH</button>
\ No newline at end of file
+<div class="form-group">
+	<div class="col-sm-offset-2 col-sm-4">
+		<button id='search-btn' type="submit" class="btn btn-primary">Search</button>
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-buttons.hbs]: missing "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs [hunk 4]
```diff
@@ -1,16 +1,27 @@
 <div class="form-group">
-	<label for="name">Name</label>
-	<input type="text" class="form-control" id="name"  name="name" placeholder="Enter an asset name">
+	<label for="name" class="custom-form-label col-lg-2 col-md-2 col-sm-12 col-xs-12 text-right">Name :</label>
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Missing "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs [hunk 5]
```diff
@@ -1,16 +1,27 @@
...
+	<div class="custom-form-right col-lg-5 col-md-8 col-sm-8 col-xs-12">
+		<input type="text" class="form-control" id="name"  name="name" placeholder="Enter an asset name">
+	</div>
 </div>
+
 <div class="form-group">
-	<label for="mediaType">Version</label>
-	<input type="text" class="form-control" id="version" name="version" placeholder="Version Eg: 1.0.0">
+	<label for="mediaType" class="custom-form-label col-lg-2 col-md-2 col-sm-12 col-xs-12 text-right">Version :</label>
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Missing language translation related "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs [hunk 6]
```diff
@@ -1,16 +1,27 @@
...
+	<div class="custom-form-right col-lg-5 col-md-8 col-sm-8 col-xs-12">
+		<input type="text" class="form-control" id="version" name="version" placeholder="Version Eg: 1.0.0">
+	</div>
 </div>
+
 <div class="form-group">
-	<label for="mediaType">Lifecycle State</label>
-	<input type="text" class="form-control" id="lcState" name="lcState" placeholder="Lifecycle State">
+	<label for="mediaType" class="custom-form-label col-lg-2 col-md-2 col-sm-12 col-xs-12 text-right">Lifecycle State :</label>
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Missing "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs [hunk 7]
```diff
@@ -1,16 +1,27 @@
...
+	<div class="custom-form-right col-lg-5 col-md-8 col-sm-8 col-xs-12">
+		<input type="text" class="form-control" id="lcState" name="lcState" placeholder="Lifecycle State">
+	</div>
 </div>
+
 <div class="form-group">
-	<label for="mediaType">Tags</label>
-	<input type="text" class="form-control" id="tags" name="tags" placeholder="Enter a tag">
+	<label for="mediaType" class="custom-form-label col-lg-2 col-md-2 col-sm-12 col-xs-12 text-right">Tags :</label>
```
Comment (cnapagoda) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Missing "t" helper

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs (hunk 1 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs (hunk 2 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-body.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-buttons.hbs (hunk 3 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/advanced-search-buttons.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs (hunk 4 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs (hunk 5 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs (hunk 6 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

File: apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs (hunk 7 above)
Comment (Malintha) [inline apps/publisher/extensions/app/publisher-common/themes/default/partials/primary-search-attributes.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

**Pull request merged**